app/views/password_resets/show.html.haml

```diff
@@ -0,0 +1,14 @@
+%section.reset_password.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_tag password_resets_path do
+        %header
+          %h1 Reset Your Password
+        %fieldset.form-group
+          = label_tag :password, "New Password"
+          = hidden_field_tag :token, @token
+          .row
+            .col-sm-4
+              = text_field_tag :password
```

Comment:
I would strongly recommend using `password_field_tag` as with text other people can see the password. :) And you want user to have it hidden with ******.